compiler/modules/front/tests/testParser.bal

```diff
@@ -209,9 +216,6 @@ function validTokenSourceFragments() returns string[][]|error {
          ["UV", "expr", string`"\""`, string`"\""`],
          ["UV", "expr", string`"what"`, string`"what"`],
          ["UV", "expr", string`"Say \"what\" again."`, string`"Say \"what\" again."`],
-         // ref
-         ["V", "expr", "x", "x"],
```

Comment:
Why are we removing this?

Reply:
2 tests where being repeated. Noticed because now the test-cases get de-duplicated due to map.

Reply:
Oh I see. OK.